cmd/main.go

```diff
@@ -52,6 +53,9 @@ func init() {
 	utilruntime.Must(operatorv1alpha1.AddToScheme(scheme))
 	utilruntime.Must(shipwrightv1alpha1.AddToScheme(scheme))
 	//+kubebuilder:scaffold:scheme
+
+	// Fetch the namespace and store for later use
+	common.OpenShiftBuilNamespaceName = common.FetchCurrentNamespaceName()
```

Comment:
Yikes! Changing a global variable that is intended to be a constant makes code super hard to follow. This will only lead to thorny, hard to debug / "works on my cluster" bugs.

I also would not invoke `FetchCurrentNamespaceName` in the `init` function - this is reserved for stuff that should work 100% of the time in production. Since this function does a disk read, an I/O failure is possible.